experiments/AMIP/moist_mpi_earth/coupler_utils/conservation_checker.jl

```diff
@@ -129,16 +144,20 @@ function plot_global_energy(cc, coupler_sim, figname1 = "total_energy.png", fign
     diff_ρe_tot_slab = (cc.ρe_tot_land .- cc.ρe_tot_land[1])
     diff_ρe_tot_slab_seaice = (cc.ρe_tot_seaice .- cc.ρe_tot_seaice[1])
     diff_toa_net_source = (cc.toa_net_source .- cc.toa_net_source[1])
+    diff_friction_sink = (cc.friction_sink .- cc.friction_sink[1])
     diff_ρe_tot_slab_ocean = (cc.ρe_tot_ocean .- cc.ρe_tot_ocean[1])
 
     times_days = times ./ (24 * 60 * 60)
     Plots.plot(times_days, diff_ρe_tot_atmos[1:length(times_days)], label = "atmos")
     Plots.plot!(times_days, diff_ρe_tot_slab[1:length(times_days)], label = "land")
     Plots.plot!(times_days, diff_ρe_tot_slab_seaice[1:length(times_days)], label = "seaice")
     Plots.plot!(times_days, diff_toa_net_source[1:length(times_days)], label = "toa")
+    Plots.plot!(times_days, diff_friction_sink[1:length(times_days)], label = "friction")
     Plots.plot!(times_days, diff_ρe_tot_slab_ocean[1:length(times_days)], label = "ocean")
 
-    tot = cc.ρe_tot_atmos .+ cc.ρe_tot_ocean .+ cc.ρe_tot_land .+ cc.ρe_tot_seaice .+ cc.toa_net_source
+    tot =
+        cc.ρe_tot_atmos .+ cc.ρe_tot_ocean .+ cc.ρe_tot_land .+ cc.ρe_tot_seaice .+ cc.toa_net_source .+
+        cc.friction_sink
```

Comment:
I wonder if we actually don't need to take into account the friction_sink here, as at least in the atmosphere the surface drag of momentum doesn't enter the tendency of ρe_tot. (I guess kinetic energy flux is already taken into account when calculating surface flux of total energy.)

Reply:
Discussed offline - the effects of this will be investigated in the future. For now we'll ignore KE dissipation here, so as to be consintent with the total energy tendency calculation.